/*注意：1、 在处理异常的时候 catch里面不能空着 否则 你就是把异常给隐藏掉了
 * 2、catch时候 如果你知道异常的名称 就写出明确的值
 * 
 * 几个常见方法：  public String getMessage():返回消息字符串
 * */
package 异常;

public class ExceptionDemo4 {
	public static void main(String[] args) {
		
		int a=10;
		int b=0;
		
		
		try{
			System.out.println(a/b);
		}catch(ArithmeticException e){
			System.out.println("这里是经过异常处理的");
		}
		System.out.println("over");

	}

}
